src/main/cpp/kernels/SystemML.cu

```diff
@@ -106,6 +106,7 @@ __forceinline__ __device__ double binaryOp(double x, double y, int op) {
         case 12 : return max(x, y);
         case 13 : return getBoolean((int)llrint(x) & (int)llrint(y));
         case 14 : return getBoolean((int)llrint(x) | (int)llrint(y));
+	case 19 : return getBoolean(x ^ y);
```

Comment:
`llvm` installation failed due to system hardware check. So, with my next task for implementing matrix operations for `xor, !, |` I will change this file, for now will remove the change.